src/main/java/fr/greencodeinitiative/php/checks/AvoidDoubleQuoteCheck.java

```diff
@@ -30,7 +30,7 @@
 import org.sonar.plugins.php.api.visitors.PHPSubscriptionCheck;
 import org.sonarsource.analyzer.commons.annotations.DeprecatedRuleKey;
 
-@Rule(key = "EC66")
+@Rule(key = "GCI66")
```

Comment:
check python rules ... we have to keep the old one because of tracability
here you must add a new line with "@DeprecatedRuleKey(repositoryKey = "ecocode-php", ruleKey = "EC66")"

same review feedback for all other rules, please.